roles/kafka_broker/tasks/secrets_protection.yml

```diff
@@ -0,0 +1,54 @@
+---
+- name: Create Kafka Broker Config with Secrets Protection
```

Comment:
Same as above. Looks unrelated.

Reply:
Rolled back. Was a part of merge conflict as i pulled my changes from 7.1.x to 7.0.0.